toptal/picasso
# feat: [FX-1255] Rename Subheader to PageHead

[FX-1255]

### Description

Rename Subheader to PageHead in lab

### How to test

- check [demo](https://picasso.toptal.net/rename-subheader/?path=/story/layout--page), make sure Subheader is not mentioned

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[FX-1255]: https://toptal-core.atlassian.net/browse/FX-1255
